src/components/AssetAccountDetails.tsx

```diff
@@ -39,7 +39,7 @@ export const AssetAccountDetails = ({ assetId: caip19, accountId, route }: Asset
           <AssetTransactionHistory limit={3} assetId={caip19} accountId={accountId} />
         </Stack>
         <Stack flex='1 1 0%' width='full' maxWidth={{ base: 'full', xl: 'sm' }} spacing={4}>
-          <TradeCard />
+          <TradeCard defaultBuyAssetId={caip19} />
```

Comment:
Do we actually need a prop for this? We should be able to surface this from the same selector this is coming from: https://github.com/shapeshift/web/blob/develop/src/pages/Assets/Asset.tsx#L21
This way, we don't have to drill the assetId (caip19) down

Reply:
I think that using `useAsset` is cleaner, but @cjthompson commented this, for reference https://github.com/shapeshift/web/pull/1210#issuecomment-1073038330

Waiting for him then :)